zulipterminal/model.py

```diff
@@ -1086,6 +1088,23 @@ def _handle_update_message_flags_event(self, event: Event) -> None:
             set_count(list(message_ids_to_mark & indexed_message_ids),
                       self.controller, -1)
 
+    def _handle_update_emoji_event(self, event: Event) -> None:
+        """
+        Handle update of emoji
+        """
+        realm_emoji_update = event['realm_emoji']
+        for emoji in realm_emoji_update:
+            if realm_emoji_update[emoji]['deactivated']:
+                # Removes emojis that have been deleted
+                # from the active_emoji_data
+                _ = self.active_emoji_data.pop(
+                    realm_emoji_update[emoji]['name'], "")
```

Comment:
Let's adjust this loop to avoid repeated indexing of `realm_emoji_update[emoji]` - note that you're only cycling through the keys of the dict right now.